sirepo_bluesky/tests/test_sirepo_flyer.py

```diff
@@ -13,7 +13,7 @@ def test_smoke_sirepo():
 
 @vcr.use_cassette('sirepo_bluesky/tests/vcr_cassettes/test_sirepo_flyer.yml')
 def test_sirepo_flyer():
-    from ..re_config import RE, db, ROOT_DIR
+    from ..re_config import RE, ROOT_DIR  # , db

```

Comment:
Maybe we don't need to package `re_config.py` at all, as it normally comes from the IPython startup files.

In the tests, it can just be directly defined in the tests, as well as the `ROOT_DIR`.